packages/frontend-2/components/workspace/header/Header.vue

```diff
@@ -189,17 +201,17 @@ const actionsItems = computed<LayoutMenuItem[][]>(() => [
   [
     ...(isMobile.value
       ? [
-          ...(isWorkspaceAdmin.value
-            ? [{ title: 'Move projects', id: ActionTypes.MoveProjects }]
-            : []),
+          { title: 'Settings', id: ActionTypes.Settings },
```

Comment:
I know it was already like this, but recently we've been adding ... after menu items that toggle a dialog, maybe add them here for all 3?

Reply:
Yeah I also went back and forth on this. But would you also add it in the account dropdown then? I don't think it feels natural to put "..." after Settings there. It feels more natural to add "..." when there's another step to an action to make it clear to the user that it doesn't trigger immediately (such as "Leave workspace..."). And opening the Settings page is not an "action".
![CleanShot 2024-10-15 at 15 11 06@2x](https://github.com/user-attachments/assets/10da3663-52dc-4bb8-9eec-716870afbd6f)

I don't have a good way to explain this yet :D.

Reply:
I think that could work